proxy.cert

```diff
@@ -0,0 +1,144 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I am not sure it is good practice to make the certificate public on gitHub.
Please clarify the usage of this file.

Reply:
Thanks for noticing. 

This is a file that was left over from an earlier version of the code. It is not necessary nor do I think will work anymore if used. I will take it out and push the changes.
The certificate needed for some of the functions to work should be placed in one's own `private/` directory